metrics/histogram.go

```diff
@@ -1,6 +1,6 @@
 package metrics
 
-// Histograms calculate distribution statistics from a series of int64 values.
+// Histogram methods calculate distribution statistics from a series of int64 values.
```

Comment:
Please don't add `methods` all over, it's a simple interface.

Reply:
Thank you - updated.